### DISCUSSION TITLE
Problem in learning, problem in chapter 6

### DISCUSSION THREAD

**Discussion by TheRebellionDreamer** [vercel/next.js#57805] [App Router]
Hi, I'm having a problem with applying sids in my App router training. Here is the error

<img width="990" alt="Снимок экрана 2023-10-31 в 13 35 28" src="https://github.com/vercel/next.js/assets/86956786/368940ef-0cc6-4b7b-b6c9-9326946b486e">

In the base itself 3 tables are created instead of 4. I'm not very good with databases, so thanks in advance!

**Comment by chopsueey** [comment]
I have the exact same problem.

Edit: It works for me now, yet I don't know why. I hope it works for you as well.
